flexbox/src/main/java/com/google/android/flexbox/FlexboxLayoutManager.java

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.flexbox;
+
+import android.content.Context;
+import android.support.v4.view.ViewCompat;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.AttributeSet;
+
+/**
+ * LayoutManager for the {@link RecyclerView}. This class is intended to be used within a
+ * {@link RecyclerView} and offers the same capabilities of measure/layout its children
+ * as the {@link FlexboxLayout}.
+ */
+public class FlexboxLayoutManager extends RecyclerView.LayoutManager {
+
+    private RecyclerView mRecyclerView;
```

Comment:
do not keep a reference to the RV. In theory, you should not need it and directly accessing it may trigger issues